### PR TITLE
bump to hippo-openapi 0.9, hippo v0.14.1

### PR DESCRIPTION
**Pull request by bacongobbler**
This makes spin compatible with Hippo v0.18.1. Backwards compatibility with older versions of Hippo (i.e. the ones that are deployed via the fermyon installer) is not guaranteed.

### CODE REVIEW

**Review by fibonacci1729 — APPROVED** [review]

**Comment by bacongobbler** [comment]
I'll let y'all handle the merging of this PR - once merged we should move quickly to cut a new release of Spin and update fermyon/installer to bump spin/hippo

**Comment by itowlson** [comment]
Does this also make it safe to merge #611 or should that wait until the installer is updated?

**Comment by bacongobbler** [comment]
Ooh, good catch! Yes, #611 could be merged at the same time. Hippo 0.18.1 implements the status code checks.

**Comment by bacongobbler** [comment]
As a heads up, a new release of Hippo will soon come out with changes to the API endpoints. Someone on the team pointed out that the API endpoints were not following REST best practices, and we figured it'd be better to break the API now rather than later.

As a result, a new version of hippo-openapi (0.10) will be released soon accommodating those changes. I'd suggest holding off on a spin release until that version is out.

**Comment by bacongobbler** [comment]
closing in favour of https://github.com/fermyon/spin/pull/678.

**Pull request closed without merging**
